src/index.ts

```diff
@@ -67,7 +73,45 @@ class OtplessManager {
     const phoneHintLibResult = await OtplessInstance.showPhoneHintLib(showFallback)
     return phoneHintLibResult
   }
+
+  async attachSecureSDK(appId: string) {
+    return await OtplessInstance.attachSecureSDK({ appId });
+  }
+}
+
+// Singleton class to prevent multiple listeners from being created.
+class OtplessSimUtils {
+  private static instance: OtplessSimUtils | null = null;
+  private simStatusChangeListener: any
+
+  private constructor() {}
+
+  clearListeners() {
+    if (this.simStatusChangeListener) {
+      this.simStatusChangeListener.remove();
+      this.simStatusChangeListener = null;
```

Comment:
add 
`await OtplessInstance.setSimEjectionsListener({ isToAttach: false });`
keep it out of if